pkg/authenticator/config/config.go

```diff
@@ -143,5 +144,17 @@ func populateConfig() (*Config, error) {
 		defaultConfig.ClientCertPath = envVal
 	}
 
+	defaultConfig.ContainerMode = DefaultContainerMode
+	// If CONTAINER_MODE is defined in the env we take its value
+	if envVal := os.Getenv("CONTAINER_MODE"); envVal != "" {
+		defaultConfig.ContainerMode = os.Getenv("CONTAINER_MODE")
+	}
+
+	defaultConfig.SupportMultipleApplications = DefaultSupportMultipleApplications
+	// If SUPPORT_MULTIPLE_APPS is defined in the env we take its value
+	if envVal := os.Getenv("SUPPORT_MULTIPLE_APPS"); envVal != "" {
+		defaultConfig.SupportMultipleApplications = os.Getenv("SUPPORT_MULTIPLE_APPS")
```

Comment:
in your PR description you say we support `CONTAINER_MODE` OR `SUPPORT_MULTIPLE_APPS`, but there is no logic here to only allow one or the other. does this code represent the flow we'd want?